src/setupTests.ts

```diff
@@ -23,22 +23,22 @@ customGlobal.fetchMock = customGlobal.fetch;
 configure({ adapter: new Adapter() });
 
 jest.mock('react-router', () => ({
-  ...(jest.requireActual('react-router') as AnyObject),
+  // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+  // @ts-ignore: expected ts type error
+  ...jest.requireActual('react-router'),
```

Comment:
Ignoring the code comments and just looking at the code change, the `as AnyObject` part has disappeared. Although `AnyObject` sounds quite generic and I'm not sure how much it defines the type, my actual question is: is this change losing some type checks that Typescript could provide? If so, that doesn't sound like a good direction.

Reply:
This is a special case in tests, because return value of the "jest.requireActual" is type "unknown". The spread operator ("..."), where the return value is used, does not accept object of that type. Casting it as AnyObject fixes that error, but raises another error: "This assertion is unnecessary since it does not change the type of the expression."

Instead of type casting and then forcing type casting error to disappear with @ts-ignore, it now ignores the first linting error. 

That type casting is useless anyway, because it changes type of an argument used in a call, not the type of an object used anywhere else. "as AnyObject" was added to prevent one linting error, but that created new error with new linter rules.